packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -62,6 +62,7 @@
   "Accounts_OAuth_Custom_Token_Path": "Token Path",
   "Accounts_OAuth_Custom_Token_Sent_Via": "Token Sent Via",
   "Accounts_OAuth_Custom_Username_Field": "Username field",
+	"Accounts_OAuth_Dolphin_Url": "Dolphin URL",
```

Comment:
Removed, also changed the setting name to match the pattern used instead of starting with API_